share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -36,6 +38,16 @@ else()
     endfunction()
 endif()
 
+macro(add_rocm_core_dependency)
```

Comment:
This is a macro to keep the linter happy. It was objecting to the complexity of the code.

Reply:
It should be prefixed with `rocm_package_`.